src/main/java/com/toofifty/easyblastfurnace/methods/Method.java

```diff
@@ -21,8 +21,8 @@
     protected final MethodStep withdrawAdamantiteOre = new ItemStep(ItemID.ADAMANTITE_ORE, "Withdraw adamantite ore");
     protected final MethodStep withdrawRuniteOre = new ItemStep(ItemID.RUNITE_ORE, "Withdraw runite ore");
 
-    protected final MethodStep withdrawIceGloves = new ItemStep(ItemID.ICE_GLOVES, "Withdraw ice gloves");
-    protected final MethodStep equipIceGloves = new ItemStep(ItemID.ICE_GLOVES, "Equip ice gloves");
+    protected final MethodStep withdrawIceOrSmithsGloves = new ItemStep(ItemID.ICE_GLOVES, "Withdraw ice gloves or smiths gloves (i)");
+    protected final MethodStep equipIceOrSmithsGloves = new ItemStep(ItemID.ICE_GLOVES, "Equip ice gloves or smiths gloves (i)");
```

Comment:
```suggestion
    protected final MethodStep withdrawIceOrSmithsGloves = new ItemStep(ItemID.ICE_GLOVES, ItemID.SMITHS_GLOVES_I, "Withdraw ice gloves or smiths gloves (i)");
    protected final MethodStep equipIceOrSmithsGloves = new ItemStep(ItemID.ICE_GLOVES, ItemID.SMITHS_GLOVES_I, "Equip ice gloves or smiths gloves (i)");
```

This'll allow it to overlay/highlight both items

Reply:
Thank you, not sure how I missed that